setup.py

```diff
@@ -234,12 +236,29 @@ def run(self):
 
 requirements_file = "requirements.txt"
 
-if '--enable_training' in sys.argv:
+local_version = None
+if parse_arg_remove_boolean(sys.argv, '--enable_training'):
     packages.extend(['onnxruntime.training',
                      'onnxruntime.training.amp',
                      'onnxruntime.training.optim'])
-    sys.argv.remove('--enable_training')
     requirements_file = "requirements-training.txt"
+    # with training, we want to follow this naming convention:
+    # stable:
+    # onnxruntime-1.7.0+cu11.1.training-cp36-cp36m-linux_x86_64.whl
```

Comment:
>11.1 [](start = 26, length = 4)

can we get rid of the dot, similar to PyTorch

Reply:
fixed